src/Components/Dropdowns/DropdownButton.tsx

```diff
@@ -90,6 +93,9 @@ export const DropdownButton = forwardRef<
         <span className="cf-dropdown--selected">{children}</span>
         {status === ComponentStatus.Loading ? (
           <TechnoSpinner diameterPixels={20} />
+        ) : !!trailingIcon ? (
```

Comment:
I now see why you did the above though - if we set `trailingIcon` to a default, we still need to adjust the CSS style.

Reply:
Yes! one of the classnames has a animation that we do not want for our vertical caret.